Remotion/Web/Development.WebTesting.IntegrationTests.Infrastructure/CustomWebTestConfigurationFactory.cs

```diff
@@ -103,5 +118,12 @@ private string GetBinaryPath (string localBrowserDirectory, string binaryName)
     {
       return Path.Combine (localBrowserDirectory, binaryName + ".exe");
     }
+
+    private IInternetExplorerConfiguration CreateInternetExplorerConfiguration ([NotNull] WebTestConfigurationSection configSettings)
+    {
+      ArgumentUtility.CheckNotNull ("configSettings", configSettings);

```

Comment:
argcheck is not needed on private member and the attribute is probably not helpful either.

Reply:
Actually, we might just inline this method?